pkg/api/v1/types_swagger_doc.go

```diff
@@ -0,0 +1,1164 @@
+/*
```

Comment:
Yes we can. It is just that I didn't see anywhere else this convention. Do you want me to change the name to: types_swagger_doc _generated.go?